meson.build

```diff
@@ -58,6 +58,10 @@ if brotli_found_all
   args += '-DCPPHTTPLIB_BROTLI_SUPPORT'
 endif
 
+if host_machine.system() == 'darwin'
+  deps += dependency('appleframeworks', modules: ['CoreFoundation', 'Security'])
+endif
```

Comment:
I didn't notice this earlier, but putting the `dependency()` call here would make the `CoreFoundation` and `Security` frameworks required even when not using building with TLS support. Could you please move these into the `if openssl_dep.found()` block? Thanks again :)

Reply:
fixed